.github/workflows/integration-tests.yml

```diff
@@ -0,0 +1,90 @@
+name: Gemini Integrations Tests
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name:
+        uses: actions/checkout@v4
+      - name: Setup Go
+        uses: actions/setup-go@v5
+        with:
+          go-version: "1.23"
+      - name: Build
+        run: make build
+      - uses: actions/upload-artifact@v4
+        with:
+          name: bin
+          path: "./bin/gemini"
+          if-no-files-found: error
+          retention-days: 1
+  test:
+    needs: [build]
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        gemini-features: ["basic", "normal", "all"]
+        gemini-concurrency: [16]
+        oracle-scylla-version: ["5.2.19"]
+        test-scylla-version: ["5.2.19"]
+      fail-fast: false
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/download-artifact@v4
+      - name: Start ScyllaDB
+        id: scylla
+        run: |
+          SCYLLA_TEST_VERSION=${{ matrix.test-scylla-version }} \
+          SCYLLA_ORACLE_VERSION=${{ matrix.oracle-scylla-version }} \
+            docker compose -f scripts/docker-compose-scylla.yml up -d || exit 1
+
+          until docker logs gemini-oracle 2>&1 | grep "Starting listening for CQL clients" > /dev/null; do sleep 1; done
+
+          until docker logs gemini-test 2>&1 | grep "Starting listening for CQL clients" > /dev/null; do sleep 1; done
+
+          ORACLE_IP=$(docker inspect --format='{{ .NetworkSettings.Networks.gemini.IPAddress }}' gemini-oracle)
+          TEST_IP=$(docker inspect --format='{{ .NetworkSettings.Networks.gemini.IPAddress }}' gemini-test)
+
+          echo "ORACLE_IP=$ORACLE_IP" >> $GITHUB_ENV
+          echo "TEST_IP=$TEST_IP" >> $GITHUB_ENV
```

Comment:
Can you please move all these steps to the `Makefile`

Reply:
We already have some scirpts, for running the environment, I did not use them, is it ok if I remove them and put setup phase in `Makefile`?

Reply:
Please do it